packages/create/src/build.config.ts

```diff
@@ -35,7 +41,27 @@ export async function before({ project }: Project.Context) {
             if (!entrypoint && filename.indexOf("/") === -1) {
                 entrypoint = filename;
             }
-            await cp(file, createPkg.resolve("dist/templates/", name, filename));
+            const source = await readFile(file, "utf-8");
+
+            for (const [, pkgName] of source.matchAll(/from "(@[^/"]+\/[^/"]+|[^/"]+)[^"]*"/g)) {
```

Comment:
nit, maybe to be more specific also add a space at the beginning and a semicolon at the end?

Reply:
and does that work with "@matter/main/types" ? because would try to add this package as dep in L51? I think you need to cut a bit more or?

Reply:
Addressed the nit and added comment referencing half-ass nature.  Careful or I'll end up doing it correctly with typescript api and this code will get much uglier. ;)

R.e. second question, fine as is, it only extracts prefixes of the form "foo" or "@foo/bar".